integration-tests/js-compute/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "scripts": {
     "install:ci": "npm ci",
-    "build:test": "base=$npm_config_test; webpack \"./fixtures/$base/$base.ts\" --output-path \"./fixtures/$base\" --output-filename \"$base.js\"; ../../target/release/js-compute-runtime \"./fixtures/$base/$base.js\" \"./fixtures/$base/$base.wasm\"",
+    "build:test": "base=$npm_config_test; webpack \"./fixtures/$base/$base.ts\" --output-path \"./fixtures/$base\" --output-filename \"$base.js\"; true",
```

Comment:
Two questions:

1. Why not move the webpack call to the shell script too? I'm not sure there's any value in the `build:test` command if most of the work is done in a wrapper anyway.

2. What are the "webpack failures" in question here? I see this was in a commit related to CompressionStream which is currently a draft API we've omitted from `index.d.ts`. In general I think it'd be good to know if our tests are badly typed; that's likely either a bug in the test or in the types. So if this is a question of letting webpack fail on type errors, maybe we should do that and fix our handling of CompressionStream another way. But that can be a separate issue/PR; it shouldn't block this one.

Reply:
> 1. Why not move the webpack call to the shell script too? I'm not sure there's any value in the `build:test` command if most of the work is done in a wrapper anyway.

Currently `webpack` is managed by npm, which is why it's run via the script in  `package.json`. We could expose a new `webpack` script that would forward all args to the executable managed by npm, which would make it easier to call directly from the `build-one.sh` script.

> 2. What are the "webpack failures" in question here? I see this was in a commit related to CompressionStream which is currently a draft API we've omitted from `index.d.ts`. In general I think it'd be good to know if our tests are badly typed; that's likely either a bug in the test or in the types. So if this is a question of letting webpack fail on type errors, maybe we should do that and fix our handling of CompressionStream another way. But that can be a separate issue/PR; it shouldn't block this one.

The webpack failures are because we're referring to a class that's not defined: `CompressionStream`. Currently the reference in `extend-from-builtins` causes all tests to fail to build unless we ignore failures from the webpack script. The existing script in `package.json` was inadvertently ignoring errors from webpack by separating the calls to `webpack` and `js-compute-runtime` with `;` instead of `&&`, which is why this isn't causing problems on the main branch.

I think the right fix here is to add types for `CompressionStream`, but I agree that we should do that in a separate PR.